lib/uninstall.js

```diff
@@ -16,9 +16,14 @@ class Uninstall extends BaseCommand {
     return 'uninstall'
   }
 
+  /* istanbul ignore next - see test/lib/load-all-commands.js */
+  static get params () {
+    return ['save']
+  }
+
   /* istanbul ignore next - see test/lib/load-all-commands.js */
   static get usage () {
-    return ['[<@scope>/]<pkg>[@<version>]... [-S|--save|--no-save]']
+    return ['[<@scope>/]<pkg>[@<version>]...']
```

Comment:
Ooh, this is a bug getting shaken out.  `npm uninstall` doesn't actually do anything with the `@version` part, hasn't for a _very_ long time, afaict.  We should drop that from the usage.